### PR TITLE
File Added

### PR DESCRIPTION
**Pull request by cloudexpert8899**
File Added

### CODE REVIEW

File: text.txt [hunk 1]
```diff
@@ -1,5 +1 @@
-This is my test file.
-This is my test file.
-This is my test file.
-This is my test file.
-This is my test file.
+This is my test file. sdf sf
```
Comment (devendravannela) [inline text.txt:1]: This is not the correct code. remove repeatable code

File: mytest.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+sdfsdfsdf dsf sf ds
```
Comment (devendravannela) [inline mytest.txt:1]: dont use nested loop

**Review by devendravannela — CHANGES_REQUESTED** [review]

File: text.txt (hunk 1 above)
Comment (cloudexpert8899) [reply text.txt:1]: Done!

File: mytest.txt (hunk 2 above)
Comment (cloudexpert8899) [reply mytest.txt:1]: done!

**Review by devendravannela — APPROVED** [review]

**Pull request merged**